tests/unit/model/relationships-test.js

```diff
@@ -66,3 +66,43 @@ test('eachRelatedType() iterates over relations without duplication', function(a
 
   assert.deepEqual(relations, ['occupation', 'person']);
 });
+
+test('normalizing belongsTo relationship names', function(assert) {
+  const UserProfile = DS.Model.extend();
+
+  let User = DS.Model.extend({
+    userProfile: DS.belongsTo(),
```

Comment:
same comment as below, this should assert and I'm surprised it does not, UserProfile needs the inverse defined.

Reply:
Ditto here, I added the inverse relationship definition to `UserProfile` above.